jobs/config.json

```diff
@@ -2763,6 +2763,12 @@
     ], 
     "scenario": "kubernetes_e2e"
   }, 
+  "ci-perf-tests-kubemark-100-benchmark": {
+    "args": [
+      "./benchmark/runner.sh"
```

Comment:
This should not use docker. This script already runs inside a container so it seems unnecessary to have the container launch another container.